src/Commands/AzureBestPractices/AzureBestPracticesGetCommand.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using AzureMcp.Arguments;
+using AzureMcp.Models.Command;
+using Microsoft.Extensions.Logging;
+using ModelContextProtocol.Server;
+
+
+namespace AzureMcp.Commands.BestPractices;
+
+public sealed class AzureBestPracticesGetCommand(ILogger<AzureBestPracticesGetCommand> logger) : BaseCommand()
```

Comment:
Add Client live test, to invoke this tool and validate the best practices data is returned.